newIDE/app/src/Hints/index.js

```diff
@@ -53,7 +53,15 @@ export const getExtraObjectsInformation = (): {
   'BitmapText::BitmapTextObject': [
     {
       kind: 'warning',
-      message: t`This object is experimental and not yet complete. It might have bugs or incomplete support in GDevelop.`,
+      message: t`This object is experimental and not yet complete. It might have bugs or incomplete support in GDevelop, be sure to read the wiki by clicking on help button bellow.`,
+    },
+    {
+      kind: 'info',
+      message: t`For a pixel type font, you must disable the Smooth checkbox related to your texture in the game resources to disable anti-aliasing.`,
+    },
+    {
+      kind: 'info',
+      message: t`For editing the font size you have to make a new export in an external editor like bmFont, click on help button bellow for more help.`,
```

Comment:
The font size is stored directly inside the font. If you want to change it, export again your font using an external editor like bmFont. Click on the help button to learn more.